frontend/src/components/SourcesTableCard/SourcesTableCard.tsx

```diff
@@ -131,12 +132,17 @@ export const SourcesTableCard: FunctionComponent<SourcesTableCardProps> = (props
                 value={frozenQuery}
                 options={dropDownValues}
                 onChange={onFilterByFrozenDataSource}
+                data-testid="sources-dropdown-filter"
               />
             </Col>
           </Row>
         </Card.Body>
         <Card.Body>
-          <SourcesTable sources={props.sources} activeSource={props.activeSource} />
+          <SourcesTable
+            sources={props.sources}
+            activeSource={props.activeSource}
+            value={frozenQuery}
```

Comment:
Could we give this prop a different name, value kind of suggests an input perhaps name it `sourceFilterValue` just for more context